### PR TITLE
Added JeffreWay to people list.

### PR DESCRIPTION
**Pull request by sh6210**
### Added Jeffre Way to people list.

* Repositioned JSON elements
* Added Jeffre Way to people list

### CODE REVIEW

File: src/data/people.json [hunk 1]
```diff
@@ -1,23 +1,23 @@
 [
     {
-        "title": "Taylor Otwell",
-        "known_for": "Creator of Laravel",
+        "title": "Jeffrey Way",
```
Comment (me-shaon) [inline src/data/people.json]: Please add new entry at the bottom of the list, not at the top of the list.

File: src/data/people.json [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+        "known_for": "Developer of Laravel",
```
Comment (me-shaon) [inline src/data/people.json]: This is wrong. He is not `Developer of Laravel`. It should be `Creator of Laracasts`.

**Review by me-shaon — CHANGES_REQUESTED** [review]

File: src/data/people.json (hunk 1 above)
Comment (sh6210) [reply src/data/people.json]: sure.

File: src/data/people.json (hunk 2 above)
Comment (sh6210) [reply src/data/people.json]: sorry, fixed the typo.

File: src/data/people.json [hunk 3]
```diff
@@ -22,7 +22,7 @@
         ]
     },
     {
-        "title": "Mohamed Said",
+        "title": "- Mohamed Said",
```
Comment (me-shaon) [inline src/data/people.json]: Please fix this typo.

File: src/data/people.json [hunk 4]
```diff
@@ -42,5 +42,27 @@
                 "link": "https://www.youtube.com/themsaid"
             }
         ]
+    },
+    {
+        "title": "Jeffrey Way",
+        "known_for": "Creator of Laracast",
```
Comment (me-shaon) [inline src/data/people.json]: It should be `Laracasts`, not `Laracast`. :)

**Review by me-shaon — CHANGES_REQUESTED** [review]
Thanks for your contribution. There are some small typos still need to be fixed. I'll merge the PR as soon as you fix them. :)

File: src/data/people.json (hunk 3 above)
Comment (sh6210) [reply src/data/people.json]: Fixed.

File: src/data/people.json (hunk 4 above)
Comment (sh6210) [reply src/data/people.json]: Fixed.

**Review by me-shaon — APPROVED** [review]

**Pull request merged**